target/linux/qualcommax/ipq807x/base-files/etc/init.d/smp_affinity

```diff
@@ -0,0 +1,36 @@
+#!/bin/sh /etc/rc.common
+
+START=99
```

Comment:
The startup priority should be below 95 that is "done" meant to be the last  and runs e.g. /etc/rc.local that it supposed to be the last. This could be 93 or something. (There is a race to-be-the-last or -not-important and too many things are already past 95)

As this has no dependencies to other services starting first, this could easily be 50 or 60 or whatever. There is no need to be the last.

But in any case, the script would add performance to qualcommax routers.

Reply:
What would be your recommendation?

My logic was that this doesn't depend on anything so it might as well just run last

Reply:
Wonder if it's possible to set the affinity before wifi and ethernet interface goes up?

Reply:
> What would be your recommendation?

At maximum 94 for "principle reasons". 
95 is designed to be "the last" for done (S95done) to run /etc/rc.local  after the services have been initialised.  (there have been too many recent services after that, but preferably there should be none.)

93 should be ok.

And I guess that even much earlier would be ok, as wifi drivers (hostapd wpad) get initialised at priority 19.
I speculated above that 50 or 60 might be ok.

Reply:
Changed to 93, that should be fine.